data/json/professions.json

```diff
@@ -5158,5 +5158,81 @@
                 "panties"
             ]
         }
+    },
+        {
+        "type": "profession",
+        "ident": "bio_sci",
+        "name": "Mad Bionics Scientist",
+        "description": "Before all this you studied bionics. You tried your experiments on animals and yourself.", 
+        "traits": [
+            "ANIMALDISCORD",
+            "PSYCHOPATH"
+        ],
+        "skills": [
+            {"level": 4, "name": "electronics"},
+            {"level": 3, "name": "mechanics"}
+        ],
+        "points": 4,
+        "CBMs": [
+            "bio_alarm",
+            "bio_geiger",
+            "bio_armor_head",
+            "bio_batteries"
+        ],                
+        "entries": [
+            { "item": "goggles_nv", "charges": 50}
+        ],
+        "items": {
+            "both":[
+               "glasses_reading",
```

Comment:
glasses_reading are for short-sighted characters. Short-sighted characters get them automatically.
There is no benefit to wearing glasses with 20/20 eyesight, so no point in having them.

Reply:
You find them in labs generally. I just thought it might be a good idea.